src/Components/CreateImageWizard/formComponents/Oscap.tsx

```diff
@@ -88,14 +107,43 @@ const ProfileSelector = ({ input }) => {
     reinitDependingSteps(change);
   };
 
-  const handleSelect = (_, selection) => {
+  const handleSelect = (_: React.MouseEvent, selection: string) => {
     setProfile(selection);
     setIsOpen(false);
     change(input.name, selection);
     reinitDependingSteps(change);
     change('file-system-config-radio', 'manual');
   };
 
+  const options = [
+    <OScapNoneOption setProfileName={setProfileName} key="oscap-none-option" />,
+  ];
+  if (isSuccess) {
+    options.concat(
+      profiles.map((profile_id) => {
+        return (
+          <OScapSelectOption
+            key={profile_id}
+            profile_id={profile_id}
+            setProfileName={setProfileName}
+          />
+        );
+      })
+    );
+  }
+
+  if (isFetching) {
+    options.push(
+      <SelectOption
+        isNoResultsOption={true}
+        data-testid="policies-loading"
+        key={'None'}
+      >
+        <Spinner size="md" />
+      </SelectOption>
+    );
+  }
+
```

Comment:
@lavocatt , I moved this part outside of Select, becuase children of select should be React.ReactElement[] and if its in the middle of the select it can return boolean